src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -81,117 +112,278 @@ public void focus() {
+                """);
+
+        contentMap.put("Clearing All Entries", """
+                Clearing All Entries\n
+                Clears all entries from the FART book.\n
+                Format: `clear`\n
+                This command will remove all contacts from the FART book.
+                """);
+
+        contentMap.put("Marking a Person as Paid/Unpaid", """
+                Marking a Person as Paid/Unpaid\n
+                Marks the specified person from the FART book as paid/unpaid.\n
+                Format: `paid INDEX` / `unpaid INDEX`\n
+                Marks the person at the specified INDEX as paid/unpaid.
+                The INDEX refers to the index number shown in the displayed person list.
+                The index must be a positive integer (1, 2, 3, …​).\n
+                Examples:
+                `list` followed by `paid 2` marks the 2nd person in the FART book as paid.
+                `find Betsy` followed by `paid 1` marks the 1st person in the results of the find command as paid.
+                `list` followed by `unpaid 2` marks the 2nd person in the FART book as unpaid.
+                `find Betsy` followed by `unpaid 1` marks the 1st person in the results of the find command as unpaid.
+                """);
+
+
+        contentMap.put("Saving and Editing Data", """
+                Saving and Editing Data\n
+                FART data is automatically saved after any command.
+                Data is stored in a JSON file at `[JAR file location]/data/addressbook.json`.
+                """);
+
+        contentMap.put("Exiting the Program", """
+                Exiting the Program\n
+                Use the `exit` command to quit the application.
+                """);
+
+        contentMap.put("Command Summary", """
+                Command Summary\n
+                | Action | Format | Examples |
+                |--------|--------|----------|
+                | Add    | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS b/BIRTHDAY [t/TAG]…​` | `add n/James Ho 
+                p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 b/11 09 2001 t/friend t/colleague` |
+                | Clear  | `clear`     | - |
+                | Delete | `delete INDEX` | `delete 3` |
+                | Edit   | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​` | `edit 2 n/James Lee 
+                e/jameslee@example.com` |
+                | Find   | `find KEYWORD [MORE_KEYWORDS]` | `find James Jake` |
+                | List   | `list`      | - |
+                | Help   | `help`      | - |
+                | Paid   | `paid INDEX` | `paid 3` |
+                | Unpaid | `unpaid INDEX` | `unpaid 3` |
+                """);
+
+        initializeCommandSummaryData();
+
+        updateTextFlow(contentMap.get("Introduction"));
+
+        tocListView.getSelectionModel().selectedItemProperty().addListener((observable, oldValue, newValue) -> {
+            if (newValue != null) {
```

Comment:
Is there another way to circumvent this somewhat "deep" nesting?

Reply:
happy to inform you that the deep nesting has been circumvented